orchestrator/orchestrator.go

```diff
@@ -49,7 +51,10 @@ func NewOrchestrator(log log.Logger, closeApp context.CancelCauseFunc, networkCo
 	// Sping up OpSim to fornt the L2 instances
 	for i := range networkConfig.L2Configs {
 		cfg := networkConfig.L2Configs[i]
-		l2OpSims[cfg.ChainID] = opsimulator.New(log, closeApp, nextL2Port, l1Anvil, l2Anvils[cfg.ChainID], l2Anvils, networkConfig.InteropDelay)
+		if cfg.Host == "" {
+			cfg.Host = defaultHost
+		}
```

Comment:
right now the CLI flags are not passed to the L2 chains. I think you need to pass something like

```
cfg.Host = networkConfig.L2Host
```